Project21/Project21/intlist.c

```diff
@@ -0,0 +1,212 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 2 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// c file for creating a linked list
+// ------------------------------------------------------------------------------------------------
+#include"intlist.h"
+#include<stdlib.h>
+#include<stdio.h>
+#include<stdbool.h>
+#include <malloc.h>
+
+bool deleted;// Boolean varaible to keep track of deleted list 
+
+/// <summary>Creates a new LinkedList to hold integers</summary>
+/// <returns>pointer to linked list</returns>
+LinkedList* Create () {
+   LinkedList* list = (LinkedList*)malloc (sizeof (LinkedList));
+   if (list != NULL) {
+      list->head = NULL;
+      list->deleted = false;
+   }
+   return list;
+}
+
+Node* CreateNode (int data) { //creates a new node with specified value
+   Node* newNode = (Node*)malloc (sizeof (Node));
+   if (newNode != NULL) {
+      newNode->data = data;
+      newNode->next = NULL;
+   }
+   return newNode;
+}
+
+/// <summary>Adds element to the end of the list</summary>
+/// <param name="data"></param>
+/// <returns>Error code (if any)</returns>
+int Add (LinkedList* list, int data) {
+   //checks if the list is deleted

```

Comment:
This comment is stating the obvious. There's no need to add this. 
For Documentation comments (starting with /// ) there's no need to mention params unless you need to explain what a certain parameter is.